views/partials/account-navigation.ejs

```diff
@@ -5,10 +5,15 @@
       src="<%-profileData.profileImg.thumbnail%>"
       onerror="this.onerror=null; this.src='<%-defaultProfileImg%>'"
     />
-    <div class="profile-name">
-      <%-profileData?.displayName || profileData?.username %>
+    <div class="profile-name-div">
+      <div class="profile-name">
+        <%-profileData?.displayName || profileData?.username %>
+      </div>
+      <img
+        id="user-country"
+        src="../../icons/flags/<%-profileData.country%>.png"
```

Comment:
What happens if user's country code does not match any file names in the flags folder?